src/check/README.md

```diff
@@ -0,0 +1,11 @@
+<p align="center">
+    <img src="../../image/logo.svg" height="200" alt="Mamba logo"/>
+</p>
+
+# Check
```

Comment:
Add constraint generation stage. 
Also specify that errors can happen at any stage.